docs/source/features/mixed_precision.rst

```diff
@@ -27,25 +68,46 @@ NVIDIA H100 GPU introduced support for a new datatype, FP8 (8-bit floating point
 
    * - Argument
      - Description
-   * - transformer_engine
-     - TE and related functionality can be enabled by setting this boolean argument to True. If this argument is not set to True, all subsequent arguments will be ignored.
    * - fp8
-     - Enables FP8 training. For transformer networks, the QKV, projection, FC1, and FC2 matrix multiplications are executed using the fourth-generation NVIDIA H100 Tensor Cores with FP8 support.
-   * - fp8_e4m3
-     - Training recipe format for FP8. Activations, weights, and gradient tensors use the E4M3 format.
-   * - fp8_hybrid
-     - Training recipe format for FP8. Activations and weight tensors use the E4M3 format, whereas gradient use the E5M2 format to satisfy the additional dynamic range requirement for backward tensors. This is the default setting.
+     - Training recipe format for FP8: 'hybrid' or 'e4m3'. 'hybrid' is set by default. With 'hybrid', activations and weight tensors use the E4M3 format, whereas gradient use the E5M2 format to satisfy the additional dynamic range requirement for backward tensors.
```

Comment:
suggested revision

The training recipe format for FP8 can be set to either 'hybrid' or 'e4m3', with 'hybrid' being the default. In the 'hybrid' format, activations and weight tensors use the E4M3 format, while gradients use the E5M2 format to meet the additional dynamic range requirements for backward tensors.